drivers/qemu/driver.go

```diff
@@ -429,18 +427,27 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 
 	// Parse configuration arguments
 	// Create the base arguments
-	accelerator := "tcg"
+	emulator := "qemu-system-x86_64"
+	if driverConfig.Emulator != "" {
+		emulator = driverConfig.Emulator
+
+	}
+	accelerator := ""
```

Comment:
Is there a backward compatibility concern we're going to need to deal with by removing the default? i.e. does QEMU fall back to something reasonable here?